src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py

```diff
@@ -214,8 +441,36 @@ def process_shift(self, events, shift_name):
         smu = event_muon[event_level]
         smu = smu[:, 0]
         sjets = event_jet[event_level]
+
+        # Filter jets with btagDeepFlavC > 0.6
+        jet_c_sel_hist = sjets.btagDeepFlavC > 0.6
```

Comment:
can you comment where hte c-jet efficiency is ~70%

Reply:
Commented